Sources/MapboxCoreNavigation/EHorizon/RoadGraphPosition.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import MapboxNavigationNative
+
+extension RoadGraph {
+
+    /** The position of a point object in the road graph. */
+    public struct Position {
+
+        /** The edge identifier that forms the point object. */
+        public let edgeIdentifier: ElectronicHorizon.Edge.Identifier
+
+        /** The distance from the start of the point object to the user’s location as a fraction of the point object’s length from 0 to 1. */
+        public let fractionFromStart: Double
+
+        init(_ native: GraphPosition) {
+            self.edgeIdentifier = UInt(native.edgeId)
+            self.fractionFromStart = native.percentAlong
```

Comment:
What does it mean for a point object to have a percentage along the object?

Reply:
Edges are "long" objects which have length > 0, but point objects are just points on graph. So to define position of some point-like object on graph we use combination of edge id + "percent along". "Percent along" is a float number in 0 - 1 range which defines offset from the beginning of the edge to the exact position of point. Also I'd say that "fraction" is probably better term here than "percent" since it is in 0 - 1 range, but not 0 - 100. We use "percent" to be consistent with Valhalla which historically uses this term.